syntax = "proto3";

option csharp_namespace = "GrpcNetworking";

service DataStorage {
  rpc SavePlayer(PlayerData) returns (SaveResponse) {}
  rpc LoadPlayer(LoadRequest) returns (PlayerData) {}
}

message PlayerData {
  int32 round = 1;
  bytes serialized_player_data = 2;
}

message LoadRequest {
  int32 round = 1;
}

message SaveResponse {
}
